composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "riari/laravel-forum",
+    "name": "seka19/laravel-forum",
```

Comment:
@seka19 wtf

Reply:
I'm sorry, I made another commit to use the package in my project, without waiting for the acceptance of the pullrequest. I did not know that the second commit would also go into the pullrequest.

I will now make a new right pullrequest.